### ISSUE TITLE
Help Command

### ISSUE THREAD

**Issue by ghost** [SpigotMC/BungeeCord#187]
I also think there should be some sort of help command that displays the default bungee commands.

**Comment by darthandrew1** [comment]
Sorry to bump this, but it wouldn't be that hard and I definitely second this. I assume it'd be a console-only command.

**Comment by DeepRobin** [comment]
Yeah, it is an idea. But i think it should a module because some bungeecord server owner dont want a help command
